src/main/java/nl/topicus/mssql2monetdb/CopyToolConfig.java

```diff
@@ -877,6 +877,10 @@ else if (key.equals("uselockedmode"))
 			{
 				table.setUseLockedMode(boolValue);
 			}
+			else if (key.equals("allowempty"))
```

Comment:
moet het altijd lowercase zijn? Zou het niet beter zijn dit met equalIgnoreCase of zo te doen?